samples/test/sample.test.js

```diff
@@ -49,11 +56,14 @@ const delay = async test => {
 describe('quickstart sample tests', () => {
   before(async () => {
     await bucket.create();
+    await bigquery.createDataset(datasetId, options);
```

Comment:
I'm worried this might leak datasets over time, we can use this approach to avoid it:

https://github.com/googleapis/nodejs-bigquery/blob/master/samples/test/datasets.test.js#L124

Reply:
It's deleted in line66 now.